### PR TITLE
1735: Update meetup FAQ page

### PR DESCRIPTION
**Pull request by KyleBlankRollins**
This PR addresses #1735. Specifically, it includes general edits for style consistency and clarity and adds more benefits to the **Meetup organizer benefits** section.

### CODE REVIEW

**Comment by KyleBlankRollins** [comment]
Looks like I don't have the ability to add labels for Writing Day. @rosewms, #1735 says to tag you for review. Could you add the Writing Day labels for this PR?

File: docs/organizer-guide/meetups/faq-meetups.rst [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Meetups FAQ
 ===========
 
-The following list of questions and answers cover some issues that you might
+The following questions and answers cover some situations that you might
 encounter when you want to `start a meetup <https://www.writethedocs.org/organizer-guide/meetups/starting/>`_.
 
-Can Write the Docs cover the Meetup.com fees?
+Does Write the Docs cover the Meetup.com fees?
 ---------------------------------------------
```
Comment (plaindocs) [inline docs/organizer-guide/meetups/faq-meetups.rst]:
```suggestion
----------------------------------------------
```

Comment (plaindocs) [reply docs/organizer-guide/meetups/faq-meetups.rst]: These are extremely annoying.

Comment (KyleBlankRollins) [reply docs/organizer-guide/meetups/faq-meetups.rst]: Ah, thanks for catching that! I haven't worked with reST much recently and forgot the heading lines need to match length.

**Review by plaindocs — APPROVED** [review]
Nice one - thanks @KyleBlankRollins

**Pull request merged**